SU-HSDO/suhumsci
# SH-5972: Use default Drupal layouts in Dashboard

# READY FOR REVIEW

## Summary
Update logic in hs_layouts.module to use default Drupal layouts in Dashboards instead of the onces used in the FE

## Urgency
medium

## Steps to Test
1. Go to https://pr1697-pksvk1y2mlhnkwglqkecukewc9y0oft0.tugboatqa.com/admin/structure/dashboard/experimental_dashboard/preview
2. Edit layout
3. Add a section and confirm there are only four layouts (1 Col, 2 Cols, 3 Cols, 4 Cols) that can be used in Dasboard
4. Add any of them and confirm they look good (there will another ticket to make it looks better, once there's a design)
5. Go to any page in the site (for example /accordions/accordions-secondary)
6. Confirm the same custom layouts can be used and not the default by Drupal

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208840953610430
